waiter/integration/waiter/basic_test.clj

```diff
@@ -684,6 +684,12 @@
           (let [response (make-request waiter-url (str "/apps/" service-id) :method :delete)]
             (assert-response-status response http-404-not-found)))
 
+        (testing "delete service with non integer timeout (should get 400)"
+          (let [timeout "Invalid-timeout-value"
+                {:keys [body] :as response} (make-request waiter-url (str "/apps/" service-id) :method :delete :query-params (str "timeout=" timeout))]
+            (assert-response-status response http-400-bad-request)
+            (is (re-find (re-pattern timeout) body))))
```

Comment:
Nitpick: please include `(str response)` as the second argument to the `is` assertion.